### ISSUE TITLE
Command to install ns?

### ISSUE THREAD

**Issue by sebj54** [NativeScript/docs-new#20]
Hi! 

I am just installing NativeScript on a fresh npm installation and I can't find how to install nativescript in the docs, especially in the introduction.

[This page](https://docs.nativescript.org/introduction.html) should show the command, before the "Overview of NativeScript development" section.

Thanks!

**Comment by rigor789** [comment]
It's right there under the `Environment Setup` section: https://docs.nativescript.org/introduction.html#environment-setup

> You will want to setup your development machine using the Environment Setup guide here which involves preparing your machine for iOS and Android development as well as installing the NativeScript CLI via npm install -g nativescript

**Comment by sebj54** [comment]
You're right! I visited the page but I was looking for a code block (or something bigger than an inline code snippet). 

Sorry for the useless issue, you may close it if appropriate.

**Comment by rigor789** [comment]
I consider this an issue, if it's not immediately obvious - means there's room for improvement... I'm a little hesitant about adding a code-block however, the Environment Setup guide has multiple steps for multiple systems for getting everything set-up, pulling out just the `npm i -g nativescript` into a code block would cause issues if all the other steps were to be ignored (XCode, Android Studio etc.) - essentially causing more frustration.

Any input/thoughts welcome what we could do to improve 🥳

**Comment by sebj54** [comment]
Maybe this command should be duplicated in the "environment setup" guides, even if it will be redundant? When I looked for the command, I clicked the first link (which leads to environment setup). I don't know why I did not click the Nativescript CLI link though.

Maybe a little reminder is also needed in the "Overview of NativeScript development" section. Something like: "Be sure to have Nativescript CLI installed. If you haven't, run `npm i -g nativescript`.

Maybe a dedicated section will be more obvious (just after "Prerequisites"). It could be named "Nativescript CLI" for instance.

**Comment by NathanWalker** [comment]
Agree here - the intro can be improved with better sequence of understanding. Thanks for feedback @sebj54

**Comment by vallemar** [comment]
I think it would be convenient to put this in a code block, although this is wrong, normally when you start with something like that you go directly to the code blocks of the documentation. @rigor789 You mean that in order to do npm install -g nativescript you need to have the jdk before, xcode doesn't? if so ... I wouldn't know the best approach for this either

**Comment by sebj54** [comment]
Actually, it can be installed before or after without failing. It will fail after onyl if you try to build or prepare a project after. Correct me if I'm wrong, but I'm sure that you can also create a project from a template without installing anything else (JDK, Android SDK, xcode)

I think `nativescript` should be installed before installing the prerequisites because `ns` provides the very useful `ns doctor` command. 

The docs could show this command very early so that users can use this command after each step if they want. Like this, they will know that the component they just installed is working and recognized by NativeScript.

**Comment by vallemar** [comment]
@sebj54 It seems to me a good flow for the documents, install nativescript, launch ns doctor and make the relevant options.  also after ns doctor put the link as a note that takes you to the installation of these components.  and after the link to leave the documents.  I think that flow in documents would be clearer and easier for new people
